numpy/lib/function_base.py

```diff
@@ -3900,7 +3900,41 @@ def percentile(a, q, axis=None, out=None,
     array([ 7.,  2.])
     >>> assert not np.all(a == b)
 
+    Selected methods for percentile estimation and their use in popular
```

Comment:
These should go in the Notes section above.  I'd remove the section underline and just have a bulleted list, preceding each method with an asterisk, as in:

```
* Method A: p(n+1): cited for example by Altman1 and CLSI2 ...
```